File: Factorial-100/factorial-100.py
#!/bin/env python

def factorial( value ) :
  result = long( 1 )
  for iteration in range( 1, value + 1 ) :
    result = result * iteration

  return result

if __name__ == '__main__' :

  for i in range( 0, 50000 + 1 ) :
    for value in range( 0, 100 + 1 ) :
       result = factorial( value )
       print result

File: Factorial-100/factorial-100.rb
#!/bin/env ruby

def factorial( value )
  result = 1
  for iteration in 1 .. value
    result = result * iteration
  end

  return result
end

if __FILE__ == $0

  for i in 0 .. 50_000
    for value in 0 .. 100
      result = factorial( value )
      puts result
    end
  end

end